### ISSUE TITLE
cannot build Docker frontend due to error from nodejs

### ISSUE THREAD

**Issue by sanposhiho** [kubernetes-sigs/kube-scheduler-simulator#38]
This issue is due to https://github.com/webpack/webpack/issues/14532

```
#15 5.142 Error: error:0308010C:digital envelope routines::unsupported
#15 5.142     at new Hash (node:internal/crypto/hash:67:19)
#15 5.142     at Object.createHash (node:crypto:130:10)
#15 5.142     at module.exports (/app/node_modules/webpack/lib/util/createHash.js:135:53)
#15 5.142     at NormalModule._initBuildHash (/app/node_modules/webpack/lib/NormalModule.js:417:16)
#15 5.142     at handleParseError (/app/node_modules/webpack/lib/NormalModule.js:471:10)
#15 5.142     at /app/node_modules/webpack/lib/NormalModule.js:503:5
#15 5.142     at /app/node_modules/webpack/lib/NormalModule.js:358:12
#15 5.142     at /app/node_modules/loader-runner/lib/LoaderRunner.js:373:3
#15 5.142     at iterateNormalLoaders (/app/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
#15 5.142     at Array.<anonymous> (/app/node_modules/loader-runner/lib/LoaderRunner.js:205:4)
#15 5.142     at Storage.finished (/app/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:55:16)
#15 5.142     at /app/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:91:9
#15 5.142     at /app/node_modules/graceful-fs/graceful-fs.js:123:16
#15 5.142     at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3) {
#15 5.142   opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
#15 5.142   library: 'digital envelope routines',
#15 5.142   reason: 'unsupported',
#15 5.142   code: 'ERR_OSSL_EVP_UNSUPPORTED'
#15 5.142 }
```

## How to solve

I have confirmed that the solution described in [the issue](https://github.com/webpack/webpack/issues/14532) will solve the problem.

**Comment by sanposhiho** [comment]
Adding `ENV NODE_OPTIONS --openssl-legacy-provider` like this will solve this issue.

![スクリーンショット 2021-11-27 9 38 07](https://user-images.githubusercontent.com/44139130/143662669-3f689bab-a6c7-4fa3-842f-6a2507be4e8e.png)

**Comment by sanposhiho** [comment]
This should be solved by #45
Thanks @tk3fftk 
/close
